common/src/HttpContentExtensions.cs

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.Devices.Shared
+{
+    internal static class HttpContentExtensions
```

Comment:
meaningful doc comment for future code readers, please

Reply:
sure, i'll add